packages/react-i18n/src/i18n.ts

```diff
@@ -481,22 +489,30 @@ export class I18n {
   // In those cases, we return the full currency code without stripping the country.
   private getShortCurrencySymbol(currencyCode = this.defaultCurrency || '') {
     const currency = currencyCode || this.defaultCurrency || '';
+
     const regionCode = currency.substring(0, 2);
-    const info = this.getCurrencySymbol(currency);
+    const info = this.getCurrencySymbol(currency, {
+      currencyDisplay: 'narrowSymbol',
```

Comment:
https://caniuse.com/?search=currencyDisplay suggests that `currencyDisplay: 'narrowSymbol'`is only supported in iOS 14.5 and greater. However the minimum version of iOS that [web currently supports is 12.2, and may be bumped to 13.0](https://github.com/Shopify/web/issues/75258), thanks to old point-of-sale devices rendering content from the web codebase in webviews.

I don't think we can use this without wrapping it in some kind of try/catch with a fallback for those old browsers.

Reply:
Will add testing for this now.